src/WorkItemMigrator/JiraExport/JiraMapper.cs

```diff
@@ -288,7 +288,7 @@ private List<WiField> MapFields(JiraRevision r)
                                 value = IfChanged<string>(item.Source, isCustomField, MapUser);
                                 break;
                             case "MapSprint":
-                                value = IfChanged<string>(_jiraProvider.Settings.SprintField, isCustomField, MapSprint);
+                                value = IfChanged<string>(item.Source, isCustomField, MapSprint);
```

Comment:
Was this intentional? Related to previous feature?

Reply:
Yes, intentional, was a bug making the sprint value not working because the jiraprovider already had the custom id for other usage. Now it´s as it should be :)